lib/licensed/sources/gradle.rb

```diff
@@ -122,61 +76,121 @@ def configurations
         end
       end
 
-      def self.add_gradle_license_report_plugins_block(gradle_build_file)
+      # Prefixes the gradle command with the project name for multi-build projects.
+      def format_command(command)
+        if config.source_path != config.pwd
```

Comment:
This function is still relevant, here is an example of when: 

Let's say my project has the following structure:

```
project
│   settings.gradle
│   build.gradle
│   licensed.yml   
│
└──/app
  │   build.gradle
```

And the licensed.yml file content is:
```
sources:
  gradle: true
apps:
  - source_path: ./app
 ```

Then since the config file is at the root of the project and I specified the apps source path, the condition`config.source_path != config.pwd` is true. 

The gradle command will be run in the subproject context like so `app:<command>`

Does this makes sense to you ?

Reply:
> Does this makes sense to you ?

Let me resay what I think you're explaining and let me know if I got it right?

When a command for the `app` subproject is run from the `project` directory, the gradle command needs to be prefixed with the project name.  Is that correct?  So to run a command on the `app` subproject while in the `project` directory, you'd need to run `app:<command>` instead of just `command`?

If so, I think I get the purpose but the usage might not be correct based on how licensed works elsewhere in the app code.  Licensed [changes the current directory](https://github.com/github/licensed/blob/bdfad9f55009596f6660d059a1c570c5919a28d5/lib/licensed/commands/command.rb#L95) to an app's `source_path` when evaluating that app.  Also, config.pwd is a [shim](https://github.com/github/licensed/blob/bdfad9f55009596f6660d059a1c570c5919a28d5/lib/licensed/configuration.rb#L57-L59) around Pathname.pwd and doesn't return the directory containing the licensed configuration file.  Unless you're manually changing the current directory, `config.source_path == config.pwd` for the duration of enumerating and evaluating dependencies for an app.

It looks like Gradle::Runner#run changes the directory to it's `@root_path` value, which is set at the creation of `gradle_runner` to `config.pwd`.  The value of `config.pwd` at that point should be equivalent to `config.source_path` though, so it's probably not actually changing the directory.

I could definitely have missed something here or misunderstood the intentions of prefixing the command with the project name though 🤷 .

Reply:
You are right, I applied the necessary changes

Thanks for taking the time to explain!